LEAF_Request_Portal/admin/templates/mod_templates_email.tpl

```diff
@@ -1189,6 +1558,14 @@
                                 saveMergedChangesToFile(fileParentFile, mergedContent);
                             });
 
+                            function toggleWordWrap() {
+                                var lineWrapping = codeEditor.editor().getOption(
```

Comment:
Should this be a let?  And why is this on two lines?

Reply:
When I save the file I notice that it formarts the file and sometimes the code splits like that. I will fix that and the var.